doc/development_process/release_process.rst

```diff
@@ -284,6 +285,20 @@ governing board.
 Processes to achieve selected certification will be determined by the Security and
 Safety Working Groups and coordinated with the TSC.
 
+Point releases
+==============
+
+As described :ref:`above <releases>`, the Zephyr Project may create follow-up
+releases from a minor release in the format [Major].[Minor].[Patch Level]. For
+example, following the v3.0 release, the project may create a v3.0.1 release
+with additional patches.
+
+No formal process exists for when these releases are created. They may be
```

Comment:
There was a proposal in the security WG to have a release cadence for supported branches (read: dot releases), it was not finalized but the text below is not in line with what discussed. @ceolin  Can we please close on this in the security WG and come up with a proposal, both for regular releases and LTS?

Reply:
Yes, I'll bring it up again.

Reply:
There is no mention to vulnerabilities here, and vulnerabilities fixes should trigger dot releases. I'll review it with the group.

Reply:
Who decides if there are vulnerability fixes? This implies a process from the security WG to the release manager for the branch, right? We need to write that down